CMakeLists.txt

```diff
@@ -23,7 +23,10 @@ include(KDECompilerSettings NO_POLICY_SCOPE)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-find_package(Qt5 REQUIRED COMPONENTS Core)
+# global Qt version, used in all find_package calls
+set(OC_QT_VERSION 5.15 CACHE STRING "Qt version to build against" FORCE)
+
+find_package(Qt5 ${OC_QT_VERSION} REQUIRED COMPONENTS Core)
```

Comment:
This will break the build of the dolphin plugin which requires the system Qt.

Reply:
Indeed. I'll revert that change.

Reply:
Actually, I think you want to make sure everyone uses the same Qt version. We can do a separate search with another version in the Dolphin plugin project, which can be built independently anyway, but is included by default for convenience reasons. But when building as part of the whole project, we should make sure to use the same version everywhere.

Reply:
I think the correct solution would be to move the plugin out of the main repo.
However it would massively increase the management overhead.

Reply:
Define "correct". Yes, it's theoretically independent ... but what do we gain from splitting it out except from overhead?

Reply:
Is there any advantage of building this plugin along with the rest of the binaries? It will most likely not work anyway unless built against the system Qt. We need to link the rest against the craft Qt, though. And I'm pretty sure we cannot ensure CMake uses the system Qt for this subproject only.